include/swift/AST/Decl.h

```diff
@@ -3587,7 +3590,8 @@ class ClassDecl final : public NominalTypeDecl {
 public:
   ClassDecl(SourceLoc ClassLoc, Identifier Name, SourceLoc NameLoc,
             ArrayRef<TypeLoc> Inherited,
-            GenericParamList *GenericParams, DeclContext *DC);
+            GenericParamList *GenericParams, DeclContext *DC,
+            bool isActor=false);
```

Comment:
It might be worth dropping the default argument so we don't forget to pass it somewhere in the future.

Reply:
Dropping it got a little nastier than intended. The unit tests have a `makeNominal` that will construct something based on a template argument. That thing is either a `ClassDecl` or a `StructDecl`.

Ended up with this little specialization of `makeNominal`. Not great, but not terrible.
```c++
  template <typename Nominal>
  Nominal *makeNominal(StringRef name,
                       GenericParamList *genericParams = nullptr) {

    auto result = new (Ctx) Nominal(SourceLoc(), Ctx.getIdentifier(name),
                                    SourceLoc(), /*inherited*/{},
                                    genericParams, FileForLookups);
    result->setAccess(AccessLevel::Internal);
    return result;
  }

  template <>
  swift::ClassDecl *makeNominal(StringRef name,
                                GenericParamList *genericParams) {
    auto result = new (Ctx) ClassDecl(SourceLoc(), Ctx.getIdentifier(name),
                                      SourceLoc(), /*inherited*/{},
                                      genericParams, FileForLookups,
                                      /*isActor*/false);
    result->setAccess(AccessLevel::Internal);
    return result;
  }
```